WeScan/Common/Error.swift

```diff
@@ -32,8 +30,6 @@ extension ImageScannerControllerError: LocalizedError {
             return "Could not setup input device."
         case .capture:
             return "Could not capture pitcure."
-        case .noRectangle:
-            return "Tried capturing while no rectangle were detected."
```

Comment:
Because zero is plural, "rectangle" can be modified to "rectangles," which is both correct and idiomatic: "while no rectangles were detected."

Another typo, "pitcure" vs "picture."

Reply:
Good catches @mrluc! Thanks for chiming in!